integration-tests/smallrye-context-propagation/src/test/java/io/quarkus/context/test/ContextEndpoint.java

```diff
@@ -72,8 +62,11 @@ public CompletionStage<String> resteasyThreadContextTest(@Context UriInfo uriInf
     @Path("/servlet")
     public CompletionStage<String> servletTest(@Context UriInfo uriInfo) {
         CompletableFuture<String> ret = all.completedFuture("OK");
+        RequestBean instance = Arc.container().instance(RequestBean.class).get();
+        String previousValue = instance.callMe();
         return ret.thenApplyAsync(text -> {
-            servletRequest.getContentType();
```

Comment:
I *assume* the only purpose of this was to check that the req. context is active. I have therefore replaced it with a CDI req. bean invocation and assertion that they are the same.